tests/unit/workflows/test_workflows_tasks_refextract.py

```diff
@@ -80,17 +83,51 @@ def test_extract_journal_info_handles_year_an_empty_string():
     eng = MockEng()
 
     assert extract_journal_info(obj, eng) is None
-    assert validate(obj.data['publication_info'], subschema) is None
-    assert obj.data['publication_info'] == [
+
+    expected = [
         {
             'artid': '134',
             'journal_title': 'Astrophys. J.',
```

Comment:
So here it looks like it's normalizing the journal name, but it puts spaces after the `.`s nonetheless.

Reply:
I believe that this is the reason: https://github.com/inspirehep/refextract/blob/19dd41af327512c98323634e0aed78fe3cffc1ec/refextract/references/kbs/journal-titles.kb#L1306

Reply:
(This is not a problem on Labs because we regenerate `journal-titles.kb` from the data.)